CFC_DataCollector/recommender/trip.py

```diff
@@ -0,0 +1,109 @@
+#maps team provided get_cost function
+#from common.featurecalc import get_cost
+import jsonpickle
+
+
+# def unit_test():
+#     info = open("C:/Users/rzarrabi/Documents/school/junior/spring/trip/e-mission-server/CFC_DataCollector/tests/data/missing_trip", "r")
+#     t = Trip(info.read())
+#     return tt
+
+class Trip(object): 
+    #Instance parameters
+    def __init__(self, json_segment):
+        self.trip_from_json(json_segment)
+
+    def trip_from_json(self, json_seg):
```

Comment:
But that creates a new Trip class, not an E_Mission_Trip or a Canonical_Trip, right? I thought that this was supposed to be a factory method that would take in the JSON object and automatically return the correct subclass. At least, that's what Shaun had told me earlier...